### DISCUSSION TITLE
Bug or expected behavior?  - Logging in with any provider returns a JWT for that provider not a Strapi user

### DISCUSSION THREAD

**Discussion by nitewulf** [strapi/strapi#7281] [Legacy Topics]
I have implemented a Google Provider in my strapi application.  I also have a front application that hits the /connect/google endpoint.  What I end up getting back is a token from google not strapi.  I can use the token to access google resources, but not Strapi resources.

Is this the expected behavior?
Do I need to implement middleware somewhere to generate a JWT for the Strapi user and pass that back to my front end application to access private content?

**Comment by lauriejim** [comment]
Hello! We updated the docs (waiting for the PR merge to be deploy) about the provider login process.
Here is the PR, you will be able to see the process.
So it's normal, you are just at the half or the auth process.
https://github.com/strapi/strapi/pull/7132/files#diff-8434e8f791257802c2af54d99bc2f32d
